imagery/i.cca/testsuite/test_i_cca.py

```diff
@@ -0,0 +1,92 @@
+"""
+Name:      i.cca tests
+Purpose:   Test correctness of generated outputs
+
+Author:    Maris Nartiss
+Copyright: (C) 2023 by Maris Nartiss and the GRASS Development Team
+Licence:   This program is free software under the GNU General Public
+           License (>=v2). Read the file COPYING that comes with GRASS
+           for details.
+"""
+
+from grass.script.core import tempname
+
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+
+
+class OutputMatchTest(TestCase):
+    """
+    Compare values to output generated by pre-7.0 version of the module.
+    Comparison values were obtained with pre-ccmath rewrite version
+    664305e4b8de924c8dfb5385e8bf6c18fbf649be
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Ensures expected computational region and generated data"""
+        cls.use_temp_region()
+        cls.runModule("g.region", raster="lsat7_2000_20")
+        cls.group_name = tempname(10)
+        cls.subgroup_name = "vis"
+        cls.runModule(
+            "i.group",
+            group=cls.group_name,
+            subgroup=cls.subgroup_name,
+            input="lsat7_2000_20,lsat7_2000_30,lsat7_2000_40",
+        )
```

Comment:
```suggestion
        cls.runModule("g.region", raster="lsat7_2002_20")
        cls.group_name = tempname(10)
        cls.subgroup_name = "vis"
        cls.runModule(
            "i.group",
            group=cls.group_name,
            subgroup=cls.subgroup_name,
            input="lsat7_2002_20,lsat7_2002_30,lsat7_2002_40",
        )
```

There is an CI error: "b'ERROR: Raster map <lsat7_2000_20> not found\n'" - the available Landsat datasets are of 2002. Updated accordingly.

Reply:
This will probably affect the expected results, @marisn?

Reply:
Markus, I have had no time (and am not certain if my math level is up to task) to go over formulas from the original paper (or was it a technical document?) to create a synthetic dataset with known results as I did for the r.kappa module. This is the reason why this PR has not been merged as at the moment is possible only to test if module runs but not if it produces correct output.

Reply:
Sure, funny understood.
So I'd suggest to commit the data name change and see what happens next.

Reply:
So, it went further, till here:

```
 Running ./imagery/i.cca/testsuite/test_i_cca.py...
========================================================================
F
======================================================================
FAIL: test_output_values (__main__.OutputMatchTest.test_output_values)
Test correctness of i.cca output
----------------------------------------------------------------------
Traceback (most recent call last):
  File "imagery/i.cca/testsuite/test_i_cca.py", line 83, in test_output_values
    self.assertRasterMinMax(
  File "etc/python/grass/gunittest/case.py", line 539, in assertRasterMinMax
    self.fail(self._formatMessage(msg, stdmsg))
AssertionError: Wrong calculated value 
The actual maximum (14) is greater than the reference one (-3) for raster map tmp_oy5WA6oFO0.2 (with minimum -17)
```